doc/source/ray-core/runtime_env_auth.md

```diff
@@ -0,0 +1,120 @@
+(runtime-env-auth)=
+# Authenticating Remote URIs in your runtime_env
+
+This section helps you:
+
+* Avoid leaking remote URI credentials in your `runtime_env`
+* provide credentials safely in KubeRay
+* understand best practices when authenticating your remote URI
```

Comment:
```suggestion
* Understand best practices when authenticating your remote URI
```

Reply:
Should I also capitalize `provide` in the second bullet point?